proposals/solid-oidc/primer-request.seqdiag

```diff
@@ -0,0 +1,19 @@
+title: Sending a Request
+
+note over RP: 1. An AJAX request is initiated
+note over RP: 2. Creates a DPoP header token
+RP->Bob's Pod (RS): 3. Sends Request
+note over Bob's Pod (RS): 4. Checks DPoP and Access token expirations
```

Comment:
DPoP tokens do not have an expiration. They contain an `iat` claim.

Reply:
Yep. That's right.